Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -480,16 +544,38 @@ - (void)finishWithContext:(BPExecutionContext *)context {
             context.finalExitStatus = BPExitStatusAppCrashed;
             NEXT([self proceed]);
             return;
+        case BPExitStatusSimulatorDeleted:
+            self.finalExitStatus = BPExitStatusSimulatorDeleted;
+            self.exitLoop = YES;
+            return;
     }
 
 }
 
+- (void)writeDeviceIDFile {
```

Comment:
Let's do temp files for now and leave the `fork()`/IPC discussion for when we actually need a bidirectional channel.